sdk/monitor/monitor-opentelemetry-exporter/src/platform/nodejs/httpSender.ts

```diff
@@ -30,6 +31,9 @@ export class HttpSender implements Sender {
     this._appInsightsClient = new ApplicationInsightsClient({
       ...this._appInsightsClientOptions
     });
+
+    this._appInsightsClient.pipeline.removePolicy({ name: redirectPolicyName });
+    this._appInsightsClient.pipeline.addPolicy(customRedirectPolicy(), { afterPhase: "Retry" });
```

Comment:
Why are you adding a redirect policy here?, we already had code to handle the redirects before

Reply:
Removed